src/App.css

```diff
@@ -5,38 +5,42 @@
   text-align: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+.header {
+  position: relative;
+  border: 5px solid rgb(192, 5, 5);
+  border-radius: 15%;
```

Comment:
If you'd like the corners of the red outline to be less pointy you could try using `px` values for the `border-radius` value instead of `%` 🎨